sympy/geometry/parabola.py

```diff
@@ -15,6 +15,7 @@
 from sympy.geometry.line import Line, Line2D, Ray2D, Segment2D, LinearEntity3D
 from sympy.geometry.ellipse import Ellipse
 from sympy.functions import sign
+from sympy import *
```

Comment:
You should not use this, instead use a complete path to import only those things that are required. I suggest having a look at all the existing functions/classes imported to avoid flake8 errors.